editor/export/export_template_manager.h

```diff
@@ -121,10 +121,15 @@ class ExportTemplateManager : public AcceptDialog {
 	static void _bind_methods();
 
 public:
-	bool can_install_android_template();
-	Error install_android_template();
+	String get_android_build_directory(int p_export_preset_index);
+	String get_android_source_zip(int p_export_preset_index);
+	String get_android_template_identifier(int p_export_preset_index);
```

Comment:
nit: Looks like these methods can be made `static` and take a `Ref<EditorExportPreset>` argument so they can be reused by the `export_plugin` logic instead of being duplicated there.

Reply:
Done in my latest push!